test-e2e/panels/filters.test.ts

```diff
@@ -33,22 +36,51 @@ async function fourierTransformFilter(nmrium: NmriumPage) {
   ).toBeVisible();
 }
 
-async function phaseCorrectionFilter(nmrium: NmriumPage) {
-  await nmrium.clickTool('phaseCorrection');
-  await nmrium.page.fill('data-test-id=input-ph1', '-100');
-  await nmrium.page.fill('data-test-id=input-ph0', '-104');
-
-  // input debounce for 250ms
-  await nmrium.page.waitForTimeout(250);
+async function phaseCorrectionFilter(
```

Comment:
@targos I would like your opinion about where to put those functions. If I understood well your idea is to move the code as a method of NmriumPage (just the action, not the expect) ?

The number of methods (and associated code) will become quite big so I would also suggest there is one file per big method of NmriumPage instance.

Reply:
I agree with that. In this case for example I would add a method named `applyPhaseCorrection(options)` to the class.

Reply:
only applyPhaseCorrection functions? what about others?

Reply:
It's up to you. anything that may be done in more than one test can be there.

Reply:
okey perfect thanks

Reply:
> anything that may be done in more than one test can be there.

i added applyPhaseCorrection , in the next PR if i reused any other function i will add it in the class